config/edit/consent.js

```diff
@@ -51,6 +51,8 @@ if (process.env.NODE_ENV !== 'production') {
     }, // THIS IS FOR CONSENT FORMS. A DOCUMENT CAN COME FROM THE CONTEXT, OR BE AN EMBEDED LINK
   );
 }
+exports.metafields = metafields;
+
```

Comment:
is there a reason why you did this? is this a better approach?

Reply:
just wanted to avoid the this. I can revert it back if you want